### DISCUSSION TITLE
club speed error

### DISCUSSION THREAD

**Discussion by wonder99** [springbok/MLM2PRO-GSPro-Connector#40] [General]
![image](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/5347771/ad21a5f9-f4b6-43dc-96c8-a9fe81253eb6)

![image](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/5347771/0e413dc9-4ddd-4c86-8e51-ac32f461f314)

not sure why

**Comment by springbok** [comment]
I’ve been trialing a new setting on the ocr to see if it improves the results. Did it work ok before you switched to v1.01.01 which includes the putting?

**Comment by wonder99** [comment]
I think I can say yes, but I really didn't play that many holes with it.
- Paul

**Comment by wonder99** [comment]
Here's another exputt issue:
I finished a couple of holes, then when putting on the next hole, I
adjusted the ball on the matt.  It caiused the error line, and then
auto-repeated my last putt when the error was cleared.]

[image: image.png]
[image: image.png]
- Paul

**Comment by wonder99** [comment]
Another full swing failure--mlm saw it correctly.  something wrong with the
post processing of the roi capture.


[image: image.png]
[image: image.png]

**Comment by springbok** [comment]
Thanks for the feedback & testing Paul, I can't see the images are you able to reload, thanks.

**Comment by wonder99** [comment]
I think the problem with the images is that it's a reply to github. Can you
send me your direct email address and I'll forward to you

**Reply by springbok** [reply]
etiennevt@gmail.com

Back home now so will be able to look at these issues.

**Comment by springbok** [comment]
New version V1.01.02 with some bug fixes:

1. I've reverted back to the original OCR settings as the new setting was not as reliable.
2. Fixed bug that set the previous shot variable even when the current shot had an error, for example if the rapsodo timed out and generated an error due to screenshot change, when woken it would see it as a new shot and cause a ghost shot.

https://1drv.ms/u/s!Agx7aYfjrZJ8nCao7AToDgVmnfMo

**Comment by springbok** [comment]
V1.01.03:

Bug fix: Was not picking up the very first shot or putt

https://1drv.ms/u/s!Agx7aYfjrZJ8nCkcv6ZU6pdX8UO8?e=KIpedE

**Comment by springbok** [comment]
V1.01.04:

Tidy up and improve error handling

https://1drv.ms/u/s!Agx7aYfjrZJ8nXXil0kGHx-Yeh6v?e=tf1QU7
